contrib/lkt/extensions/mains/lkt_toolbox.adb

```diff
@@ -95,6 +121,36 @@ procedure Lkt_Toolbox is
       end if;
    end Print_Semantic_Result;
 
+   -------------------------------
+   -- Populate_Invalid_Decl_Map --
+   -------------------------------
+
+   function Populate_Invalid_Decl_Map
+      (Node : Analysis.LK_Node'Class) return Common.Visit_Status is
```

Comment:
We generally put `is` on its own line when there are declarations before the `begin` keyword (less confusing layout)